Sources/NIOCore/AsyncAwaitSupport+OldXcodes.swift

```diff
@@ -0,0 +1,218 @@
+//===----------------------------------------------------------------------===//
```

Comment:
This file shouldn't exist anymore, it was removed in https://github.com/apple/swift-nio/pull/2292